packages/lms/src/lib/components/navigation/Entity.svelte

```diff
@@ -31,7 +40,11 @@
   <div class="entity-inner">
     <div class="entity-header hover:bg-primary-hover-token rounded-container-token p-2">
       {#if entity.browserPath}
-        <a href={entity.browserPath} on:click={drawerClose} class="entity-title flex flex-row items-center">
+        <a
+          href={entity.browserPath}
+          on:click={drawerClose}
```

Comment:
```suggestion
          on:click={drawerClose}
          class:active={active}
```

Not tested but this is a more sveltey way

Reply:
You can just do `class:active` to be even more Sveltey